src/pages/experiments/[experimentId]/data-processing/configure-embedding/components/EmbeddingPreview.jsx

```diff
@@ -81,23 +78,51 @@ const EmbeddingPreview = () => {
     },
   };
 
+  // Start loading by fetching the settings and cell sets.
+  useEffect(() => {
+    dispatch(loadCellSets(experimentId));
+    dispatch(loadProcessingSettings(experimentId, FILTER_UUID));
+  }, []);
+
+  // Once the config is loaded, we can also load the embedding.
+  useEffect(() => {
```

Comment:
we might need an if statement in here to check if `embeddingMethod` is not defined, because if it is not, we do not want to send that dispatch request

Reply:
You're right, this is now fixed.